sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/valueentity/ValueEntity.java

```diff
@@ -22,13 +22,34 @@
 import kalix.javasdk.SideEffect;
 import kalix.javasdk.StatusCode;
 import kalix.javasdk.action.Action;
+import kalix.javasdk.eventsourcedentity.EventSourcedEntity;
 import kalix.javasdk.impl.valueentity.ValueEntityEffectImpl;
 import io.grpc.Status;
 
 import java.util.Collection;
 import java.util.Optional;
 
-/** @param <S> The type of the state for this entity. */
+/**
+ * Value Entities persist their state on every change. You can think of them as a Key-Value entity where
+ * the key is the entity id and the value is the state of the entity.
+ * <p>
+ * Kalix Value Entities have nothing in common with the domain-driven design concept of Value Objects.
+ * The Value in the name refers to the direct modification of the entity's state.
+ *
+ * When implementing a Value Entity, you first define what will be its internal state (your domain model),
+ * and the commands it will handle (mutation requests).
```

Comment:
"mutation requests" doesn't go well with the below `directly reply to the caller if the command is not requesting any state change`. Not always the request are mutation ones.

Reply:
Review to be continued.

Reply:
No more comments. The rest is a repetition of the previous or I have nothing to add.